volatility3/cli/text_renderer.py

```diff
@@ -8,8 +8,13 @@
 import string
 import sys
 from functools import wraps
-from typing import Callable, Any, List, Tuple, Dict
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Tuple
 
+from rich import box
```

Comment:
Ideally this should be in a separate file for just this renderer, and the imports in it should be protected by a `try/catch ImportError` which puts a vollog at debug level to say Rich can't be found.  It's up to you whether to have the rich renderer present, but fail to run (with a vollog warning/error) or simply not include the code if the import fails, but it should definitely be optional.